<!--yhs-->
<launch>

    <node pkg="yhs_can_control" type="yhs_can_control_node" name="yhs_can_control_node" output="screen">

	<param name="odom_frame" value="odom"/>
    <param name="base_link_frame" value="base_link"/>
    <param name="tfUsed" value="false"/>

    </node>

</launch>

